apps/chrono24/chrono24.star

```diff
@@ -0,0 +1,251 @@
+"""
+Applet: Chrono24
+Summary: Watch market performance
+Description: Gives ability to show watch market performance over multiple durations as well as seeing brand performance.
+Author: Chase Roossin
+"""
+
+load("animation.star", "animation")
+load("cache.star", "cache")
+load("encoding/json.star", "json")
+load("http.star", "http")
+load("render.star", "render")
+load("schema.star", "schema")
+
+DEFAULT_WHO = "world"
+BASE_URL = "https://www.chrono24.com/api/priceindex/performance-chart.json?type=Market&period="
+CONFIG_TIMEFRAME = "config-timeframe"
+CONFIG_VIEW_TYPE = "config-view-type"
+CONFIG_VIEW_TYPE_GRAPH = "graph"
+CONFIG_VIEW_TYPE_INDEXES = "indexes"
+CACHE_TTL = 3600  # 6 hour
+
+def main(config):
+    timeframe = config.str(CONFIG_TIMEFRAME, "_1month")
+    viewtype = config.str(CONFIG_VIEW_TYPE, CONFIG_VIEW_TYPE_GRAPH)
+
+    # Check for cached data
+    cached_data = cache.get(timeframe)
+
+    if cached_data != None:
+        print("Hit! Displaying cached data.")
+        data = json.decode(cached_data)
+    else:
+        print("Miss cache!")
+        rep = http.get(BASE_URL + timeframe)
```

Comment:
You can simplify this a bit by passing `ttl_seconds` to `http.get()`. It will cache the response accordingly. We may change the behavior of the `cache` module in the future, so using the http module's built in cache is a good choice.